k8_cortx_cloud/get-logs-from-pvc.sh

```diff
@@ -147,6 +162,16 @@ fi
 printf "Waiting for tar to complete.\n"
 kubectl exec --namespace "${namespace}" "${pod_name}" -- sh -c 'until [ -f /tmp/tarfile_created ]; do sleep 1; done' || exit_msg "Failed waiting for job to complete" 1
 
+if kubectl exec --namespace "${namespace}" "${pod_name}" -- sh -c 'ls /tmp/tar_failed &> /dev/null' 2> /dev/null
```

Comment:
I'm not sure if I get this, are you trying to check if the `tar_failed` file exists or not? If so, why not:

```suggestion
if kubectl exec --namespace "${namespace}" "${pod_name}" -- sh -c '[ -e /tmp/tar_failed ]'
```

If the file exists, the exit code is `0` which satisfies the if condition. If it doesn't exist, the exit code is `1` and the `then` block is skipped.

Reply:
Thanks! I will update.  (Yeah... not sure what I was thinking not using "-e".)  I still need the "2> /dev/null", or else I get a `command terminated with exit code 1` message, which I don't want.

FYI, This is what output looks like with 
```
k8_cortx_cloud $ ./get-logs-from-pvc.sh -s /tmp/client.namespace.cortx.3361-5node.solution.yaml cortx-ha -d baddir -f
Starting job cortx-log-cortx-ha-20220908.135349.
job.batch/cortx-log-cortx-ha-20220908.135349 created
Waiting for pod to start.
pod/cortx-log-cortx-ha-20220908.135349-rn4xx condition met
Waiting for tar to complete.
tar: cortx-ha/baddir: No such file or directory
tar: error exit delayed from previous errors
Failed to collect files from PVC cortx-ha, path=baddir
job.batch "cortx-log-cortx-ha-20220908.135349" deleted
```

Reply:
> I still need the "2> /dev/null", or else I get a `command terminated with exit code 1` message, which I don't want.

Understood. That's annoying, you can't even silence it with `--quiet`.